@stellar/design-system-website/src/App.tsx

```diff
@@ -1,34 +1,115 @@
+import { useMemo, useState, useLayoutEffect, useEffect } from "react";
 import { BrowserRouter as Router, Switch, Route } from "react-router-dom";
 import { Layout } from "@stellar/design-system";
 
-import { Details } from "components/Details";
 import { Landing } from "pages/Landing";
+import { ComponentDetails } from "pages/ComponentDetails";
+import { SideNavContext } from "context/SideNav";
 
 import "styles.scss";
 
-export const App = () => (
-  <Router>
-    {/* TODO: update project link */}
-    <Layout.Header
-      projectTitle="Design System"
-      projectLink="https://www.stellar.org"
-      hasDarkModeToggle
-    />
-
-    <Layout.Content>
-      <Switch>
-        <Route exact path="/component/:id">
-          <Details />
-        </Route>
-
-        <Route path="/">
-          <Landing />
-        </Route>
-
-        {/* TODO: add 404 */}
-      </Switch>
-    </Layout.Content>
-
-    <Layout.Footer gitHubLink="https://github.com/stellar/stellar-design-system" />
-  </Router>
-);
+const BREAKPOINT_SMALL = 600;
+const CSS_CLASS_SMALL_SCREEN = "small-screen";
+const CSS_CLASS_SIDE_NAV_OPEN_TRIGGER = "side-nav-open-trigger";
+const CSS_CLASS_SIDE_NAV_OPEN = "side-nav-open";
+const CSS_CLASS_DISABLE_BODY_SCROLL = "no-body-scroll";
```

Comment:
bummer that css vars can't be used as selector - it would've been nice to be able to just define these strings once and reuse in the scss